app/helpers/application_helper.rb

```diff
@@ -17,4 +17,12 @@ def self.regulation_code(regulation)
     regulation_id = regulation.regulation_id
     "#{regulation_id.first}#{regulation_id[3..6]}/#{regulation_id[1..2]}"
   end
+
+  # TODO: Remove this and use the original path helpers once we're removed legacy routes which clobber the correct helpers
+  def v1_api_path(resource, id = nil)
```

Comment:
Because moving the legacy routes to the bottom of the routes.rb file clobbers the helpers we actually want to use I've opted to correct these by using this application helper method.

These will automatically remedy when we've removed the legacy routes